Apps/CommonData/src/Models/EmailTemplateConfig.cs

```diff
@@ -0,0 +1,33 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Common.Data.Models
+{
+    /// <summary>
+    /// Model object representing email template configuration.
+    /// </summary>
+    public class EmailTemplateConfig
+    {
+        /// <summary>
+        /// The section key to use when binding this object.
+        /// </summary>
+        public const string ConfigurationSectionKey = "EmailTemplate";
+
+        /// <summary>
+        /// Gets the host.
+        /// </summary>
+        public string Host { get; init; } = string.Empty;
```

Comment:
NIT: Can this be required to remove the pointless initialization?

Reply:
We need to have a default value in case the configuration binding fails.

Reply:
Alternately we could throw exceptions in that case.